catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -393,6 +404,18 @@ private Double degreesToMeters(double distance) {
                 Distance.LinearUnit.KILOMETER).getAs(Distance.LinearUnit.METER);
     }
 
+    private Integer getSolrQueryRowsProperty() {
+        try {
+            Integer rows = Integer.valueOf(System.getProperty(SOLR_QUERY_ROWS_PROPERTY));
+            if (rows >= Integer.MAX_VALUE) {
```

Comment:
Should we have a less than 1 check here as well? I think a row count of 0 or less is bad.